recipes/twopaco/meta.yaml

```diff
@@ -1,27 +1,32 @@
-{% set name = "twopaco" %}
+{% set name = "TwoPaCo" %}
 {% set version = "0.9.2" %}
-{% set hash = "0493807d4ad232067058904c69a1e968682f97d62e81aa452b5fc7b64d06b044" %}
+{% set sha256 = "0493807d4ad232067058904c69a1e968682f97d62e81aa452b5fc7b64d06b044" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/medvedevgroup/TwoPaCo/archive/v{{ version }}.tar.gz
-  sha256: {{ hash }}
+  url: https://github.com/medvedevgroup/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - Makefile.patch
 
 build:
-  number: 1
+  number: 2
   skip: True  # [osx]
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - cmake
-    - tbb
-
   run:
     - tbb
+    - tbb-devel
+  host:
+    - tbb
```

Comment:
Pretty sure `tbb-devel` requires `tbb`  (actually, I think they are built from the same recipe at CF)
```suggestion
```